### DISCUSSION TITLE
“Stay Connected” button

### DISCUSSION THREAD

**Discussion by tatobarresi** [rejetto/hfs#603] [Ideas]
I associated each folder with a qrcode.
When I need to reach that directory I use the qr code.

However, each time it asks me to log in.

is there any way to keep the user logged in?

**Comment by rejetto** [comment]
there's no need for a button, because that's the default behavior.
If you use the same browser, and same user, you will only be required for login once (until HFS is restarted).
QRcode is just a link, so what you say should happen even opening 2 textual links.
The login is stored in a cookie. So be sure you use same browser and have cookies working.
